### PR TITLE
Fix tooltips max setting on game restart, ref #2010.

### PR DESCRIPTION
**Pull request by burner1024**
## Description

GUIopt.py increase only applies if you played around in Options windows.
With cold start, delay setting would pass directly to WindowManager and never get the multiplier.
This moves the final multiplier into WindowManager so it would apply on game start as well.

## Checklist

- [x] Commit messages are descriptive and explain the rationale for changes
- [x] I used the same coding style as the surrounding code 
- [x] I have tested the proposed changes
- [ ] I extended the documentation, if necessary
- [ ] The proposed change builds also on our build bots (check after submission)

### CODE REVIEW

File: gemrb/core/GUI/WindowManager.cpp [hunk 1]
```diff
@@ -41,7 +42,8 @@ Holder<Sprite2D> WindowManager::CursorMouseDown;
 
 void WindowManager::SetTooltipDelay(int delay)
 {
-	ToolTipDelay = delay;
+	// Max setting disables tooltips in originals, we just set it to an extremely high value for simplicity.
+	ToolTipDelay = (delay == 10 * Tooltip::DELAY_FACTOR) ? 1e7 : delay;
```
Comment (lynxlynxlynx) [inline gemrb/core/GUI/WindowManager.cpp:46]: Please just write the number out, which will also avoid a pointless cast.

Comment (burner1024) [reply gemrb/core/GUI/WindowManager.cpp]: Like this?

File: gemrb/core/GUI/WindowManager.cpp [hunk 2]
```diff
@@ -43,7 +42,8 @@ Holder<Sprite2D> WindowManager::CursorMouseDown;
 void WindowManager::SetTooltipDelay(int delay)
 {
 	// Max setting disables tooltips in originals, we just set it to an extremely high value for simplicity.
-	ToolTipDelay = (delay == 10 * Tooltip::DELAY_FACTOR) ? 1e7 : delay;
```
Comment (lynxlynxlynx) [inline gemrb/core/GUI/WindowManager.cpp]: No no, I meant the `1e7` double, of course named constants are fine.

Comment (burner1024) [reply gemrb/core/GUI/WindowManager.cpp]: okay

**Comment by lynxlynxlynx** [comment]
Thanks!

**Pull request merged**